diff --git a/nix/openroad.nix b/nix/openroad.nix
index 9dcf415..711daea 100644
--- a/nix/openroad.nix
+++ b/nix/openroad.nix
@@ -42,8 +42,8 @@
   bison,
   clang-tools_14,
   ioplace-parser,
-  rev ? "b16bda7e82721d10566ff7e2b68f1ff0be9f9e38",
-  sha256 ? "sha256-+JGyX81Km2XidptA3k1Y5ZPwv+4Ed39LCsPfIHWd6ac=",
+  rev ? "2c0ecdf1a04540c53c2cbaf346a783d88a6dc345",
+  sha256 ? "sha256-XJDDN3DOFwfGZ5AirLLICSEQYEgWf73zLy7rCvdeJDs=",
 }: let
   pyenv = python3.withPackages (p:
     with p; [
